.changeset/angry-cooks-clean.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/dev-process-manager": major
```

Comment:
```suggestion
"@comet/dev-process-manager": minor
```

Reply:
If we don't enforce the version then we should keep engines at >= 14 and remove the changeset altogether

Reply:
@johnnyomair FYI we do not enforce the engines but our subpackages do for example command v12 enforces nodejs >= 18 now